travis-ci/vmtest/run.sh

```diff
@@ -468,4 +474,19 @@ sudo umount "$mnt"
 
 travis_fold end shutdown
 
+tests_results["vm_tests"]=$exitstatus
+
+# Final summary - Don't use a fold, keep it visible
+echo -e "\033[1;33mTest Results:\033[0m"
+for testgroup in ${!tests_results[@]}; do
+	# Print final result for each group of tests
+	if [[ ${tests_results[$testgroup]} -eq 0 ]]; then
+		printf "%20s: \033[1;32mPASS\033[0m\n" $testgroup
+	else
+		printf "%20s: \033[1;31mFAIL\033[0m\n" $testgroup
+	fi
+	# Make exitstatus > 0 if at least one test group has failed
+	exitstatus=$((exitstatus + tests_results[$testgroup]))
```

Comment:
adding exitstatuses seems error prone and a bit confusing, maybe if + exitstatus = 1?